microsoft-edge/webview2/reference/javascript/hostobjectssyncroot.yml

```
@@ -0,0 +1,16 @@
+### YamlMime:TSType
+name: HostObjectsSyncRoot
+uid: 'WebView2Script!HostObjectsSyncRoot:class'
+package: WebView2Script!
+fullName: HostObjectsSyncRoot
+summary: >-
+  Contains synchronous proxies for all host objects added via `CoreWebView2.AddHostObjectToScript`<!-- -->.
+
+
+  If you call `coreWebView2.AddHostObjectToScript("myObject", object);` in your host code then a synchronous proxy for
+  `object` will be exposed in script as `chrome.webview.hostObjects.sync.myObject`<!-- -->.

```

```suggestion
  If you call `coreWebView2.AddHostObjectToScript("myObject", object);` in your native code, then a synchronous proxy for `object` is available to your web-side code, by using `chrome.webview.hostObjects.sync.myObject`.
```
orig. wording of external comment:
```
If you call `coreWebView2.AddHostObjectToScript("myObject", object);` in your native code then a synchronous proxy for `object` is available to your web code using `chrome.webview.hostObjects.sync.myObject`.
```